Makefile

```diff
@@ -308,6 +310,7 @@ export GO_BUILD_GCFLAGS := all=-trimpath=$(PWD)
 export GO_BUILD_FLAGS :=
 export GO_BUILD_LDFLAGS := -s -w \
     -X '$(VERSION_PATH).version=$(VERSION)' \
+-X "$(VERSION_PATH).gitCommit=$(GIT_COMMIT)" \
```

Comment:
Can you just ensure that it is alined with above?
Otherwise, it seems that we agreed with this one 👍

Reply:
Also nit: use single quotes rather than double quotes for consistency.

Reply:
Done